packages/ui5-application-writer/src/options.ts

```diff
@@ -111,8 +111,13 @@ export async function applyOptionalFeatures(
     tmplPath: string,
     ui5Configs: UI5Config[]
 ) {
+    const isEdmxProjectType = ui5App.app.projectType === 'EDMXBackend';
     if (ui5App.appOptions) {
         for (const [key, value] of Object.entries(ui5App.appOptions)) {
+            if(key === 'sapux' && !isEdmxProjectType) {
```

Comment:
Could we instead remove the option before calling this code instead of embedding the business logic directly here please? This will make the code cleaner.